### ISSUE TITLE
Is 

### ISSUE THREAD

**Issue by nekoyama32767** [aria1th/Hypernetwork-MonkeyPatch-Extension#12]

**Comment by nekoyama32767** [comment]
It is possible  to make a  totally manual mode which allows to make connection and activation function every layer?
Maybe resblock like architecture or skip connection is useful for hypernetwork

**Comment by aria1th** [comment]
Yes, but the problem is UI, I could add some way to load model but creating it is different story....
I'll try making it to save and load any arbitrary model structure, but creating it would be homework for users.

**Comment by aria1th** [comment]
https://github.com/aria1th/Hypernetwork-MonkeyPatch-Extension/tree/residual-connection

Currently skip-connection, ResBlock architecture for normal tensors are available here. 

There are few things you need to know:
![image](https://user-images.githubusercontent.com/35677394/212824272-21eebe0e-ccb0-469e-93d6-3b474afbcbc5.png)

Since it 'interpolates' attention vector, setting value lower than 1 would make loss in vector, because it cannot recover vectors from compressed form.

(It could be solved by having attention-compressing and decompressing model somehow?)

Default standard deviation is too big for attention transformation. I suggest using lower stdev.

Original webui does not support these types of hypernetworks. 
![image](https://user-images.githubusercontent.com/35677394/212824842-787f8bf0-0111-4023-842c-adeeec275c2b.png)
![image](https://user-images.githubusercontent.com/35677394/212824909-8ddb9057-1059-4574-8208-455e1f3cec05.png)

Training works well anyway, but you might have to struggle with setting.

**Comment by nekoyama32767** [comment]
Thank you for work on this. I will try it

**Comment by nekoyama32767** [comment]
It looks work well 
![aokana_side_drop-596](https://user-images.githubusercontent.com/60657593/213488627-7f7c4f3b-2b08-4c62-8e2f-19938463d205.png)
![aokana_side_drop-11920](https://user-images.githubusercontent.com/60657593/213488638-3c3ada8c-d96a-4a6c-b0cf-3bb967d7f9ed.png)

**Comment by nekoyama32767** [comment]
Skip-connection hypernetworks train on vainilla NovelAI model works excellent with a pre-trained dreambooth model
![PD5ZSKY
Z690F%4}(VC9ZQ](https://user-images.githubusercontent.com/60657593/213783367-aa6bc697-c563-475d-839a-437cf6306430.png)
